src/controllers/RoomController.ts

```diff
@@ -75,6 +75,21 @@ class RoomController implements IController {
 
   };
 
+  disableById: IRequestHandler = async (
```

Comment:
Let's not make it a separate function, this should be part of the update controller... we just be able to just send a PUT /room/id and modify isDisable